homeworks/yana.haivoronska_Yana_Haivoronska/Game Project/style.css

```diff
@@ -0,0 +1,24 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+.main {
+  max-width: 409px
+}
```

Comment:
use font-family here)

Reply:
Let's check if I make it right , sorry for disturbing.